rabbitmq/plugins.sls

```diff
@@ -0,0 +1,31 @@
+{% from "rabbitmq/map.jinja" import rabbitmq with context %}
+
+include:
+  - .configure
+  - .install
+
+{% for plugin in rabbitmq.get('plugins', []) %}
+{% if plugin.get('package_name') %}
+download_rabbitmq_plugin_{{ plugin.name }}:
+  file.managed:
+    - name: /usr/lib/rabbitmq/lib/rabbitmq_server-{{ rabbitmq.version.split('-')[0] }}/plugins/{{ plugin.package_name }}
+    - source: https://www.rabbitmq.com/community-plugins/v3.6.x/{{ plugin.package_name }}
```

Comment:
Yeah, probably worth doing that. I can add a fix unless you want to try doing a PR to patch it after we merge.